docs/Softwares/matlab.md

```diff
@@ -0,0 +1,265 @@
+# Use MATLAB on NJIT HPC
+
+## Installation steps of MATLAB on local machine
```

Comment:
Our software application pages should follow a template for consistency.

* short description of the application
* table containing the various versions and their module files (maybe a note column if a particular toolchain module needs to be loaded)
* how to use the application on our cluster
* example of a slurm script, example files or location on the cluster where the example files can be found.
* How to use application if available on Bright or Ondemand portal
* Any tips or tricks for using the application or external links that may be useful including tutorials, user guide from developer etc
* Lastly, how to use/install the application locally and connect from it to the cluster (for example this first section here should be last). 

@gwolosh anything you want to add?

Reply:
Thanks @alexpacheco for the suggestions. I see this pull request is closed, however, the files are not updated in the "main" branch. The status is "Closed with unmerged commits".